modules/scheduler/scheduler/src/main/java/org/motechproject/scheduler/service/impl/MotechSchedulerServiceImpl.java

```diff
@@ -771,6 +779,62 @@ public void unscheduleAllJobs(String jobIdPrefix) {
         return messageTimings;
     }
 
+    @Override
```

Comment:
the methods pauseJob, resumeJob, deleteJob look similar, can you put the repetitive code into one method ?